magnolia/shared/src/main/scala-2.12-2.13/zio/config/magnolia/DeriveConfig.scala

```diff
@@ -9,7 +9,7 @@ import java.time.{LocalDate, LocalDateTime, LocalTime}
 import java.util.UUID
 import scala.collection.immutable
 
-final case class DeriveConfig[T](desc: Config[T], isObject: Boolean = false) {
+final case class DeriveConfig[T](desc: Config[T], isObject: Boolean = false, constValue: Option[T] = None) {
```

Comment:
This is necessary to support `Config.Switch`, but I wonder if there is a better way, or at least replace `isObject` with `constValue.isDefined`.